src/Common/Select.php

```diff
@@ -273,7 +271,7 @@ protected function addColWithAlias($spec)
      *
      * @param string $alias The column to remove
      *
-     * @return null
+     * @return boolean
```

Comment:
bool and boolean are the same, (http://php.net/manual/en/language.types.boolean.php). Changed it to bool. The old return type of null was incorrect: every return value is either `true` or `false`.